### PR TITLE
Naturalize's suggestions

### PR DESCRIPTION
**Pull request by condda**
Naturalize's renaming suggestions to improve the codebase consistency

### CODE REVIEW

File: picasso/src/main/java/com/squareup/picasso/Picasso.java [hunk 1]
```diff
@@ -509,11 +509,11 @@ Bitmap quickMemoryCacheCheck(String key) {
   }
 
   void complete(BitmapHunter hunter) {
-    Action single = hunter.getAction();
+    Action action = hunter.getAction();
```
Comment (JakeWharton) [inline picasso/src/main/java/com/squareup/picasso/Picasso.java]: This should remain as "single". Hunters have a "single" action and also "joined" actions.

**Comment by condda** [comment]
Ok, I'll remove that commit

**Comment by JakeWharton** [comment]
Thanks!

**Comment by condda** [comment]
My pleasure

**Pull request merged**